app/apphandlers/setup.go

```diff
@@ -53,7 +53,7 @@ func rudderCoreWorkSpaceTableSetup() error {
 	return validators.CheckAndValidateWorkspaceToken()
 }
 
-func setupReadonlyDBs() (gw, rt, batchrt *jobsdb.ReadonlyHandleT, err error) {
+func setupReadonlyDBs() (gw *jobsdb.ReadonlyHandleT, err error) {
```

Comment:
We should probably change the function name as it only set's up GW Db

Reply:
In reality it is not that simple. It sets all readonly dbs but only returns gw db... bad design